README.md

```diff
@@ -1,2 +1,179 @@
-# metric-webapp
-React capstone project
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./banner.webp" alt="logo" />
+  <br/>
+
+  <h1><b>Metric-WebApp</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Metric-WEBAPP\] ](#-metric-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Metric-WEBAPP] <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+
+**[METRIC-WEBAPP]** This React capstone project (remember what they are?)is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+You will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Linters</li>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>React</li>
+    <li>Redux</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github pages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No database</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **[Use a minimalist style to build the project]**
+- **[Best practices of ES6 syntax]**
+- **[Use of React to the functionalities of the page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://obibaadoma.github.io/metric-webapp/)
```

Comment:
- [ ] As requested by the previous Code reviewer the link  you provided in the README file is not this application.I will suggest you to deploy the application using  [ Netlify](https://www.netlify.com/) as it is more simple. You can also follow [this link](https://www.geeksforgeeks.org/how-to-deploy-react-app-on-netlify-using-github/) for more details.
This is the page displayed when I clicked on the Live demo link

![Capture](https://github.com/obibaadoma/metric-webapp/assets/80458199/3f4cb5b6-5e07-49a7-96b6-00472f17a26a)

Reply:
done, deployed toNetlify

Reply:
changed deployment link